portal/views/patients.py

```diff
@@ -78,8 +78,8 @@ def render_patients_list(
             patient.current_qb = qb_status['visit_name']
             if research_study_id == EMPRO_RS_ID:
                 patient.clinician = '; '.join(
-                    (clinician_name_map.get(c.id, "<deleted>") for c in
-                     patient.clinicians)) or "<deleted>"
+                    (clinician_name_map.get(c.id, "not in map") for c in
+                     patient.clinicians)) or ""
```

Comment:
I'm having trouble squaring this with your comment:
`without a current clinician, use empty string, not "".`
Sorry, my python is weak...

Reply:
line 81, we attempt to look up the clinician id (`c.id`) in the name map, providing a default of "not in map" should the lookup fail - which should never happen.  (it was happening as we had non clinicians in the test data, and if it returns, i want a clear indication of the problem.)
line 82, if the entire clause left of the `or` is logically false, i.e. the patient had zero clinicians, we'll get back the empty string from the right hand side of the `or` expression.